internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -804,6 +804,19 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Repository'
+        filesystem:
+          type: array
+          items:
+            $ref: '#/components/schemas/Filesystem'
+    Filesystem:
+      type: object
```

Comment:
I suggest making both `mountpoint` and `min_size` (see below) required:
```
required:
  - mountpoint
  - min_size
```

That would firstly fix the compile error in the PR currently (by making these optional they end up as pointers, so we would at least need to dereference them before assigning them to the blueprint customisations, but then we would also need to handle the case where they are not set), but I also think it is more correct:

The mountpoint should always be mandatory, as without it, I don't think a filesystem entry (of the kind we support) make sense. Since the size is actually the minimum size, there is no harm in making that mandatory too, the caller could just set it to 0 and it should have the same effect as not setting it at all.